modules/mono/glue/cs_files/Color.cs

```diff
@@ -265,14 +263,14 @@ public Color Inverted()
             );
         }
 
-        public Color LinearInterpolate(Color b, float t)
+        public Color LinearInterpolate(Color c, float t)
         {
-            Color res = this;
+            var res = this;
 
-            res.r += (t * (b.r - this.r));
-            res.g += (t * (b.g - this.g));
-            res.b += (t * (b.b - this.b));
-            res.a += (t * (b.a - this.a));
+            res.r += t * (c.r - r);
+            res.g += t * (c.g - g);
+            res.b += t * (c.b - this.b);
```

Comment:
One of the points of changing the input variable to `c` was so that you can remove the `this.` in front of `b` :)

Reply:
Urg... silly me :) Removed `this.` too.